daemon/algod/api/server/v2/handlers.go

```diff
@@ -239,6 +239,16 @@ func (v2 *Handlers) AccountInformation(ctx echo.Context, address string, params
 		return internalError(ctx, err, errFailedLookingUpLedger, v2.Log)
 	}
 
+	latestBlkHdr, err := myLedger.BlockHdr(lastRound)
+	if err != nil {
+		return internalError(ctx, err, errFailedRetrievingLatestBlockHeaderStatus, v2.Log)
+	}
+
+	consensus, ok := config.Consensus[latestBlkHdr.CurrentProtocol]
+	if !ok {
+		return notFound(ctx, errors.New(errInternalFailure), "could not retrieve consensus information for current protocol", v2.Log)
```

Comment:
it is `internalError` rather than `notFound` since itis result does not depends on client's input

Reply:
👍